plugins/chilli/chilli_

```diff
@@ -0,0 +1,118 @@
+#!/bin/bash
```

Comment:
Please change /bin/bash to /bin/sh, if possible. This will ease the usage of this plugin on platforms without a default installation of bash (embedded systems, some *BSD, AIX, ...).
The only required change below seems to be the rewriite of the tests:
* replace double square brackets with single ones
* replace `==` with `=`